smt/surrogate_models/kplsk.py

```diff
@@ -23,13 +23,25 @@ def _initialize(self):
             types=(str),
         )
 
+        declare(
+            "pow_exp_power",
+            2.0,
+            desc="Power for the pow_exp kernel function",
+            types=(float),
+        )
+
+
     def _componentwise_distance(self, dx, opt=0, theta=None, return_derivative=False):
```

Comment:
The value pow_exp_power should be > 0 and =<2.

The kernel function $\phi(x) = \exp [- \theta |x|^p]$ is positive definite for a given positive $\theta$ if $0<p \leq 2$ [Corollary~3]{Schoenberg}. METRIC SPACES AND POSITIVE DEFINITE FUNCTIONS* BY I. J. SCHOENBERG

Reply:
Thank you for pointing it out. I added a check. Please let me know if any additional change is needed.

Reply:
Thanks a lot !